.NET/Microsoft.Recognizers.Text.DateTime/German/Extractors/GermanDateExtractorConfiguration.cs

```diff
@@ -102,6 +102,12 @@ public class GermanDateExtractorConfiguration : BaseOptionsConfiguration,IDateEx
             // 23/7
             new Regex(DateTimeDefinitions.DateExtractor9, RegexOptions.IgnoreCase | RegexOptions.Singleline),
 
+            //10
```

Comment:
Can you add examples in the code comment here that illustrate which cases would match this regex?

Reply:
Yes, I will do this with the next PR. I am still not fully done with the extractors, some of them will probably be deleted and others changed. DateExtractor 10 and 11 will most certainly be blended into the one or two of the other ones and removed after that.